### PR TITLE
426: improved pause handling

### PR DESCRIPTION
**Pull request by sviatosluv**
https://github.com/Buckitup-chat/chat/assets/90610934/2c19c24d-9b0b-4e8a-ab4b-15326b2de342

### CODE REVIEW

File: lib/chat_web/live/main_live/layout/uploader.ex [hunk 1]
```diff
@@ -175,10 +175,16 @@ defmodule ChatWeb.MainLive.Layout.Uploader do
           type={@type}
         />
       <% end %>
-      <div class="hidden">
+      <div id={"uploadErrors-" <> Utils.random_id()} class="hidden" phx-update="append">
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/layout/uploader.ex:178]: it is better to write helper function than show implementation details in markup

File: assets/js/upchunk_upload.js [hunk 2]
```diff
@@ -36,8 +36,8 @@ const uploadInitializer = (entry, onViewError) => {
     upload.on("progress", (e) => {
       const now = new Date().getTime()
 
-      if (!window.uploaderReorderInProgress && !upload.paused && e.detail < 100 && now - lastProgressUpdate > 1000) {
-        entry.progress(e.detail)
+      if (!window.uploaderReorderInProgress && upload.success && !upload.paused && e.detail < 100 && now - lastProgressUpdate > 1000) {
+        console.log("progress", e.detail)
```
Comment (sergey-lukianov) [inline assets/js/upchunk_upload.js:40]: is `entry.progress` the same as `console.log` ?

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: assets/js/upchunk_upload.js (hunk 2 above)
Comment (salseeg) [reply assets/js/upchunk_upload.js:40]: You are replacing one with another. Is it ok ?

File: lib/chat_web/live/main_live/layout/uploader.ex (hunk 1 above)
Comment (sviatosluv) [reply lib/chat_web/live/main_live/layout/uploader.ex]: done

File: assets/js/upchunk_upload.js (hunk 2 above)
Comment (sviatosluv) [reply assets/js/upchunk_upload.js]: fixed

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**